web/src/components/Navigation.vue

```diff
@@ -302,6 +316,45 @@ const moreOpen =
   ["converters", "status", "tags", "pcaps"].includes(route.name); // FIXME: type route
 const groupedTags = computed(() => store.groupedTags);
 const status = computed(() => store.status);
+const shiftPressed = ref(false);
+const onShiftPressed = computed(() => {
```

Comment:
this has a very bad name, it sounds like a handler so we either rename it or better replace it with a style that uses a class or so? not sure how this works but inline styles seem less good